config/types/schema.go

```diff
@@ -0,0 +1,179 @@
+package types
```

Comment:
Does it make sense to keep the most recent version in the repo if it's generated, or should we modify the build script to generate it every time?